kroxylicious-app/src/main/java/io/kroxylicious/app/Kroxylicious.java

```diff
@@ -35,13 +35,18 @@ public class Kroxylicious implements Callable<Integer> {
 
     private static final Logger LOGGER = LoggerFactory.getLogger("io.kroxylicious.proxy.StartupShutdownLogger");
     private static final String UNKNOWN = "unknown";
-    private final BiFunction<PluginFactoryRegistry, Configuration, KafkaProxy> proxyBuilder;
+    public static final String KROXYLICIOUS_ENVIRONMENT = "KROXYLICIOUS_ENVIRONMENT";
```

Comment:
KROXYLICIOUS_ENVIRONMENT is a bit vague.   I'd suggest DEVELOPMENT_MODE which defaults to false.